Library/Homebrew/aliases/alias.rb

```diff
@@ -18,7 +18,10 @@ def initialize(name, command = nil)
         @symlink = T.let(nil, T.nilable(Pathname))
 
         @command = if command&.start_with?("!", "%")
+          # `drop` is not defined on `String`
+          # rubocop:disable Performance/ArraySemiInfiniteRangeSlice
```

Comment:
This isn't supposed to be enabled, and upstream actually states:

> This cop was created due to a mistake in microbenchmark

https://docs.rubocop.org/rubocop-performance/cops_performance.html#performancearraysemiinfiniterangeslice

Given the large amount of errors in what is supposed to only have one extra cop, it seems like something's really broken here.

Reply:
Ah, good catch. I think the default config is only loaded when the cop is a plugin: https://github.com/rubocop/rubocop-performance/compare/v1.23.1...v1.24.0#diff-5a022e7952dfb32fcbdce1db566fce92f11071753c6ce2b60c09f5a0edf8b6fc

Unfortunately, I'm not able to follow the plugin migration suggested by the docs due to: https://github.com/rubocop/rubocop-rspec/issues/2048#issuecomment-2669585900

Reply:
I suggest waiting for the Rubocop update that fixes it.

Reply:
Agreed